new-frontiers-14/frontier-station-14
# Replace courthouse POI tiny fans with directional, fix up atmos setup

## About the PR

- Replaced tiny fans with directional fans at docks
- Colorized atmos distro and waste pipes
- Fixed up a scrubber and T-junction that had invalid placement
- Anchored atmos supply canisters and replaced mixer and pump with the default-on versions

## Why / Balance
Compliance, aesthetics, fixing broken stuff.

## How to test
Load up the courthouse grid, observe how you can breathe, check out those sweet nicely labeled pipes, nod in reserved approval of the directional fans which comply with the most current NT workplace safety regulations.

## Media
![atmos](https://github.com/user-attachments/assets/665862fa-964f-4781-8ffb-93d98575a8b1)
![dock1](https://github.com/user-attachments/assets/98c7bfce-500d-4129-9303-b4d607f99256)
![dock2](https://github.com/user-attachments/assets/824776e5-9ec4-4134-b841-7d6d10ad370d)
![pipecolors](https://github.com/user-attachments/assets/9790071c-fc55-4bc3-a96b-aeeebbc599eb)


## Requirements



## Breaking changes
None
